React/Base/RCTUtils.m

```diff
@@ -21,6 +21,8 @@
 #import "RCTAssert.h"
 #import "RCTLog.h"
 
+static const NSUInteger RCTMaxCachableImageCount = 100;
```

Comment:
Have there been discussions to allow app developers to adjust the cache size (or to opt for a cache based on the collective sizes of the images using [`totalCostLimit`](https://developer.apple.com/documentation/foundation/nscache/1407672-totalcostlimit?language=objc) and [`setObject:forKey:cost:`](https://developer.apple.com/documentation/foundation/nscache/1416399-setobject?language=objc) rather than just raw quantity)? Sure, they can always edit core code themselves, but this seems inconvenient.